src/java/org/apache/cassandra/tools/FieldUtil.java

```diff
@@ -38,32 +39,12 @@ public static void setInstanceUnsafe(Class<?> klass, Object v, String fieldName)
 
     private static void setInstanceUnsafeThrowing(Class<?> klass, Object v, String fieldName) throws Throwable
     {
-        Field field = klass.getDeclaredField(fieldName);
```

Comment:
duplicate logic makes it harder to support newer JDKs, so moved the duplicate logic to the existing `ReflectionUtils` class, and kept w/e was unique to this class